.circleci/config.yml

```diff
@@ -15,9 +15,9 @@ executors:
       DOCKER_CLIENT_VERSION: "17.09.0-ce"
       DOCKERFILE_PATH: "."
   buildx:
-    machine:
-      image: ubuntu-2004:current
-      resource_class: xlarge
+    docker:
+      - image: cimg/base:stable-18.04
+        user: root
```

Comment:
needed that locally otherwise docker denied permissions, don't know if we have to have it